src/cli/backup/backup.go

```diff
@@ -187,8 +190,15 @@ func genericCreateCommand(
 
 		bo, err := r.NewBackupWithLookup(ictx, discSel, ins)
 		if err != nil {
-			errs = append(errs, clues.Wrap(err, owner).WithClues(ictx))
-			Errf(ictx, "%v\n", err)
+			cerr := clues.Wrap(err, owner).WithClues(ictx)
+			errs = append(errs, cerr)
+
+			meta, err := json.Marshal(cerr.Core().Values)
+			if err != nil {
+				meta = []byte("Unable to marshal error metadata")
+			}
+
+			Errf(ictx, "%s\nMessage: %v\nMetadata:%s", "Unable to complete backup", err, meta)
```

Comment:
Still not a fan of displaying clues values to end users.  I do think it'll make the ux worse.  Maybe run it by @gmatev for a final eval?

Reply:
I see your point. Let me drop that.